src/Features/Core/Portable/CodeMapper/IMapCodeService.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Immutable;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Host;
+
+namespace Microsoft.CodeAnalysis.CodeMapping;
+
+internal interface IMapCodeService : ILanguageService
```

Comment:
this interface warrants documentation.  'code mapping' is not a clear concept.

Reply:
@genlu you can find docs on what the CodeMappers are on the original repository under docs/CodeMappers.md

Reply:
@osobeso sure.  but we need to doc our code as well.

Reply:
Will do. Note this is not intended to be used in actual product yet, just to unblock Razor's investigation for now. More work would probably be needed here